ts/docs.ts

```diff
@@ -48,49 +86,116 @@ async function docs(): Promise<void> {
         const schemaMdFile = path.join('public', `${title}.schema.md`)
         await encoda.write(articleMd, schemaMdFile)
 
-        const mdFile = path.join('schema', `${title}.md`)
-        if (await fs.pathExists(mdFile)) {
-          const article = await encoda.read(mdFile)
+        const maybeMdFile = path.join('schema', `${title}.md`)
+        const mdFileExists = await fs.pathExists(maybeMdFile)
+        const mdFile = mdFileExists ? maybeMdFile : schemaMdFile
+        const article = await encoda.read(mdFile)
+        // Don't output an HTML file if the Markdown document cannot be decoded into an article
+        if (!isArticle(article)) {
+          log.warn(
+            `docs.ts#57: cant decode '${mdFile}' as 'Article'; skipping HTML generation`
+          )
+          return
+        }
 
-          // Don't output an HTML file if the Markdown document cannot be decoded into an article
-          if (!isArticle(article)) {
-            return
-          }
+        log.debug(`schema2Article(${jsonFile}) => ${schemaMdFile}`)
+        if (!mdFileExists) {
+          stats.htmlFromYamls++
+          log.info(
+            `(${stats.htmlFromYamls})${maybeMdFile} doc not found; stubbing html from schema2Article(${mdFile})`
+          )
+        }
 
-          const processed = await encodaProcess(
+        // dom fragment to let readers know this file was generated automatically
+        const subheadingContent = !mdFileExists
+          ? [
+              {
+                type: 'Heading',
+                depth: 6,
+                content: [
+                  `note: this html file was autogenerated by 'ts/docs.ts::schema2Article()' from '${schemaMdFile}'; create documentation for it at '${maybeMdFile}'`
```

Comment:
I like the addition of this notice, but I think we can make it a little less technical and more user friendly by changing the wording slightly. What do you think?

Something along the lines of:
```suggestion
                  `Note: This documentation was autogenerated from '${schemaMdFile}'; Please help improve this page by submitting a Pull Request`
```